challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |string| string.capitalize if string.class == String }
```

Comment:
I personally don't think the `string.class == String` is necessary, since someone might use `capitalize` on a different type and `NoMethodErrors` would make sense in this context.